src/commands/mod.rs

```diff
@@ -0,0 +1,40 @@
+use clap::{ValueEnum, Parser, AppSettings};
+use std::fmt::{Display, Formatter};
+
+#[derive(Parser)]
+#[clap(global_setting(AppSettings::AllowLeadingHyphen))]
+pub struct Input {
```

Comment:
Right now the help message is really bare-bones. Can you add back the command description?
```suggestion
#[derive(Parser)]
#[clap(about, global_setting(AppSettings::AllowLeadingHyphen))]
pub struct Input {
```